ui/imports/shared/panels/ModuleWarning.qml

```diff
@@ -123,14 +128,24 @@ Item {
             id: layout
 
             spacing: 12
-            anchors.horizontalCenter: parent.horizontalCenter
-            anchors.verticalCenter: parent.verticalCenter
+            anchors.centerIn: parent
 
             StatusBaseText {
                 text: root.text
-                Layout.alignment: Qt.AlignVCenter
                 font.pixelSize: 13
+                font.weight: Font.Medium
+                anchors.verticalCenter: parent.verticalCenter
```

Comment:
Igor changed the `layout` to be `RowLayout` instead of `Row`, so all those `anchors` in the children below won't work likely, needs `Layout.alignment`

Reply:
Can look into it! At least I don't see an issue in the UI (been looking at this for the past months non-stop lol)

Reply:
Yeah, it will (most likely) be alright but you'll see some warnings like `can't use anchors inside a Layout` in the console